mlflow/R/mlflow/.run-tests.R

```diff
@@ -4,5 +4,21 @@ package <- parent_dir[grepl("mlflow_", parent_dir)]
 library(reticulate)
 use_condaenv(mlflow:::mlflow_conda_env_name())
 
-devtools::check_built(path = package, remote = TRUE, error_on = "note", args = "--no-tests")
```

Comment:
How about we just do `error_on="warning"`.
According to https://devtools.r-lib.org/reference/check.html
```
Passing R CMD check is essential if you want to submit your package to CRAN: you must not have any ERRORs or WARNINGs, and you want to ensure that there are as few NOTEs as possible.
```
So it's acceptable to have one or more notes. According to  https://stackoverflow.com/a/23831508/12165968, the maintainer note is safe to ignore.

The downside of my proposal is that we might be less alerted if other notes show up.

Reply:
In mlflow 1.21.0, one note (about an invalid URL) blocked the package **publication**.

Reply:
Got it. It's frustrating that CRAN also categorizes blocking issues as a note besides non-blocking ones...
Do we know whether disabling the CRAN incoming feasibility check will miss blocking issues? I didn't understand what exactly CRAN incoming feasibility check does and why setting the 7 days threshold resolves the issue...

Reply:
> Do we know whether disabling the CRAN incoming feasibility check will miss blocking issues?

> 


> I didn't understand what exactly CRAN incoming feasibility check does.

It performs a bunch of checks. You can take a look at this file:
https://github.com/wch/r-source/blob/4561aea946a75425ddcc8869cdb129ed5e27af97/src/library/tools/R/QC.R#L8005-L8008


> why setting the 7 days threshold resolves the issue.

```
── R CMD check results ────────────────────────────────────── mlflow 1.21.1 ────
Duration: 52.7s

❯ checking CRAN incoming feasibility ... NOTE
  Maintainer: ‘Matei Zaharia <matei@databricks.com>’
  
  Days since last update: 0

0 errors ✔ | 0 warnings ✔ | 1 note ✖
```

This note basically means `it's been 0 days since the last release. It's too early`. The code I attached indicates that the threshold is 7 days. It looks like there is no option to disable this recency check so we just disable the entire feasibility check by setting `remote` to `FALSE`.

Reply:
I see. I previously thought `checking CRAN incoming feasibility ... NOTE
  Maintainer: ‘Matei Zaharia <matei@databricks.com>’` just means a complaint about the maintainer.

So, as long as we don't release two versions within 7 days, it would run through the same checks as it was before this PR.

Reply:
Yep :)